app/src/main/java/com/imashnake/animite/dev/ext/Modifier.kt

```diff
@@ -0,0 +1,8 @@
+package com.imashnake.animite.dev.ext
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+
+val bottomNavigationBarSize = 65.dp
+fun Modifier.bottomNavigationBarPadding(): Modifier = this.padding(bottom = bottomNavigationBarSize)
```

Comment:
If you use a Scaffold, it handles this for you

Reply:
Adding this to #48, using it makes the transition to `MediaPage` a little glitchy.